website/docs/getting-started.md

```diff
@@ -0,0 +1,153 @@
+---
+sidebar_position: 3
+sidebar_label: Getting started
+---
+
+# Getting started with Discordeno
+
+<!-- TODO: Do we want to keep this info box? Or rewrite a bit this -->
+
+:::info
+Discordeno is not intended for beginners, however you can still utilize it.
+:::
+
+## Installation
+
+You can install the npm package `@discordeno/bot` and this package will take care of setting up the REST & Gateway for your bot.
+
+```bash
+npm install @discordeno/bot # Using npm
+yarn add @discordeno/bot    # Using yarn
+pnpm add @discordeno/bot    # Using pnpm
+bun add @discordeno/bot     # Using bun (as a package manager)
+```
+
+If you are using Deno you will need to still use the npm package, so you will need to use the `npm:` prefix in your imports.
+
+## Minimal bot that logins into Discord
+
+You can use `@discordeno/bot` as follows:
+
+```ts
+import { createBot, Intents } from '@discordeno/bot'
+
+const bot = createBot({
+  token: 'YOUR BOT TOKEN',
+  intents: Intents.Guilds | GuildMessages, // Or other intents that you might needs.
```

Comment:
```suggestion
  intents: Intents.Guilds | Intents.GuildMessages, // Or other intents that you might needs.
```

Reply:
Whoops, i forgot the Intents while copy pasting stuff around